product-create-search/src/BusinessLogic/CrudMysql.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+
+namespace App\BusinessLogic;
+use App\Entity\Painting;
+use App\Entity\PaintingInterface;
+use Doctrine\ORM\EntityManagerInterface;
+use phpDocumentor\Reflection\Types\Integer;
+use Symfony\Component\HttpFoundation\Request;
+
+class CrudMysql implements CrudMysqLI
+{
+    private $painting;
+    private $entityManager;
+    private $id;
+
+    public function __construct(PaintingInterface $paintingI, EntityManagerInterface $entityManagerInterface)
+    {
+        $this->painting = $paintingI;
```

Comment:
It is fine to use new for entities, since they will contain our application data , services are not allowed to created using new